src/ext/resources/legacygraphqlapi/schema.graphqls

```diff
@@ -1012,6 +1065,11 @@ type StopGeometries {
     googleEncoded: [Geometry]
 }
 
+input Coordinate {
+    lat: Float!
+    lon: Float!
```

Comment:
Avoid abbreviations like `lat` and `long`

Reply:
Good point. lat/lon are a bit on the edge as they are pretty standardized abbreviations but using the full words are better anyway.

Reply:
Done in https://github.com/opentripplanner/OpenTripPlanner/pull/5185/commits/f3a4219870faed100b53be6fddf8acdacf5747d9